### PR TITLE
Update PR template

### PR DESCRIPTION
**Pull request by bartzet**
# Problem

# Solution

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [ ] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [ ] Have new automated tests been implemented?
- [ ] Have new manual tests been written down?
- [ ] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-870-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-870-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://870-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-870-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-870-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://870-pr-onfido-sdk-ui-onfido.surge.sh

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -7,7 +7,10 @@
 ## Checklist
 _put `n/a` if item is not relevant to PR changes_
 
-- [ ] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
+- [ ] Has the CHANGELOG been updated?
+- [ ] Has the README been updated?
+- [ ] Has the RELEASE_GUIDELINES been updated?
+- [ ] Has the MIGRATION been updated in regards to the changes included in the PR?
 - [ ] Have new automated tests been implemented?
 - [ ] Have new manual tests been written down?
```
Comment (kopijunkie) [inline .github/PULL_REQUEST_TEMPLATE.md]: I've noticed that especially with screen updates/new screens and upcoming copy updates, we're also likely to change existing tests rather than just add new ones. Would be worth rephrasing these 2 checklist points while we're at it now.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -7,7 +7,10 @@
 ## Checklist
 _put `n/a` if item is not relevant to PR changes_
 
-- [ ] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
+- [ ] Has the CHANGELOG been updated?
+- [ ] Has the README been updated?
+- [ ] Has the RELEASE_GUIDELINES been updated?
+- [ ] Has the MIGRATION been updated in regards to the changes included in the PR?
```
[the remark this answers is not in the document]
Comment (bartzet) [reply .github/PULL_REQUEST_TEMPLATE.md]: We mention translations in the last point of the checklist:
`[ ] Have any new strings been translated?`

Maybe we could rephrase it to
`- [ ] Have any new strings been translated or the existing ones changed?` 

I'm just not sure if we should give examples on the checklist. It makes it a bit long.

How about?
`Has the MIGRATION doc been updated for any major/minor breaking changes in the PR?`

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-870-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-870-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://870-pr-onfido-sdk-ui-onfido.surge.sh

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (kopijunkie) [reply .github/PULL_REQUEST_TEMPLATE.md]: My bad. Didn't see that last point. 🙈 

But yeah, your suggestion sounds good.

[the remark this answers is not in the document]
Comment (bartzet) [reply .github/PULL_REQUEST_TEMPLATE.md]: Changed.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-870-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-870-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://870-pr-onfido-sdk-ui-onfido.surge.sh

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -7,7 +7,10 @@
...
 _put `n/a` if item is not relevant to PR changes_
 
-- [ ] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
-- [ ] Have new automated tests been implemented?
-- [ ] Have new manual tests been written down?
-- [ ] Have any new strings been translated?
+- [ ] Has the CHANGELOG been updated?
+- [ ] Has the README been updated?
+- [ ] Has the RELEASE_GUIDELINES been updated?
+- [ ] Has the MIGRATION doc been updated for any major/minor breaking changes in the PR?
```
Comment (stefaniacardenas) [inline .github/PULL_REQUEST_TEMPLATE.md]: I would make it more explicit on what we mean by updating the migration notes for minor changes.
```
[ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
[ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?

Comment (bartzet) [reply .github/PULL_REQUEST_TEMPLATE.md]: Are these the only reasons we update `minor`? I think there might be more reasons that we do it hence I'm not sure listing them in the checklist. Maybe we could have some external document created with the points what is causing the bump of the `patch`, `minor` and `major` version. The link could be added to the checklist after 🤔

Comment (stefaniacardenas) [reply .github/PULL_REQUEST_TEMPLATE.md]: Yes, it's the only reason. I think it would be good to have it in the checklist because it's quite uncommon to have migration notes for minor changes. We should still have a document in our wiki with the details of our migration process. And yes, if in the future we need to add more cases, we can refer to the wiki.

Comment (bartzet) [reply .github/PULL_REQUEST_TEMPLATE.md]: Makes sense. Thank you for the explanation.

Comment (bartzet) [reply .github/PULL_REQUEST_TEMPLATE.md]: Re: wiki, we already have it https://wiki.onfido.co.uk/pages/viewpage.action?spaceKey=ENG&title=SDK+Git+Workflow and there is a bit of explanation in the _Versioning_ section.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-870-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-870-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://870-pr-onfido-sdk-ui-onfido.surge.sh

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (stefaniacardenas) [reply .github/PULL_REQUEST_TEMPLATE.md]: Cool, we need to add a note about when to update the migration file and what to put in it. Can be done as a separate story. Can you delete the internal link, please? 🙏

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-870-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-870-pr-onfido-sdk-ui-onfido.surge.sh

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (bartzet) [reply .github/PULL_REQUEST_TEMPLATE.md]: I left my suggestions on when to update the migration in the _Redefine SDK API process change - versioning, documentation_ google doc that I was sharing with the Team last week on Slack.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://870-pr-onfido-sdk-ui-onfido.surge.sh
